src/test/kotlin/io/getstream/dokkasaurus/utils/StringExtensionsTest.kt

```diff
@@ -21,4 +24,25 @@ internal class StringExtensionsTest {
             complexTag.simpleScapeTags()
         )
     }
+
+    @ParameterizedTest
+    @MethodSource("generateCamelToSnakeCaseArguments")
+    fun `String should be converted from camelCase to snake-case`(input: String, expectedResult: String) {
+        Assertions.assertEquals(expectedResult, input.camelToSnakeCase())
+    }
+
+    companion object {
+        @JvmStatic
+        fun generateCamelToSnakeCaseArguments() = listOf(
+            Arguments.of("a", "a"),
+            Arguments.of("A", "a"),
+            Arguments.of("ab", "ab"),
+            Arguments.of("aB", "a-b"),
+            Arguments.of("AB", "a-b"),
+            Arguments.of("ABc", "a-bc"),
+            Arguments.of("ABC", "a-b-c"),
```

Comment:
This is kebab-case 😄  
![image](https://user-images.githubusercontent.com/12054216/119806000-18081000-bee2-11eb-98f6-d67564d47b1f.png)

Reply:
UPs, yep